pkgs/development/python-modules/bidict/default.nix

```diff
@@ -28,31 +25,37 @@ buildPythonPackage rec {
     hash = "sha256-WE0YaRT4a/byvU2pzcByuf1DfMlOpYA9i0PPrKXsS+M=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
     setuptools
-  ];
-
-  propagatedBuildInputs = [
-    sphinx
+    wheel
   ];
 
   nativeCheckInputs = [
     hypothesis
-    py
+    pytest
     pytest-xdist
     pytestCheckHook
-    pytest-benchmark
-    sortedcollections
-    sortedcontainers
     typing-extensions
   ];
 
+  pytestFlagsArray = [
+    # Pass -c /dev/null so that pytest does not use the bundled pytest.ini, which adds
+    # options to run additional integration tests that are overkill for our purposes,
+    # and (most problematically) adds the --benchmark-disable option. This option
+    # requires taking a dependency on pytest-benchmark in order to avoid `error:
+    # unrecognized arguments: --benchmark-disable ...`. Rather than taking an
+    # additional dependency on pytest-benchmark only to disable it, we can just avoid it
+    # in the first place by bypassing the bundled pytest.ini as follows.
+    "-c"
+    "/dev/null"
+  ];
+
   pythonImportsCheck = [ "bidict" ];
 
   meta = with lib; {
-    homepage = "https://github.com/jab/bidict";
-    changelog = "https://github.com/jab/bidict/blob/v${version}/CHANGELOG.rst";
-    description = "Efficient, Pythonic bidirectional map data structures and related functionality";
+    homepage = "https://bidict.readthedocs.io";
+    changelog = "https://bidict.readthedocs.io/changelog.html";
+    description = "The bidirectional mapping library for Python.";
     license = licenses.mpl20;
     maintainers = with maintainers; [ jakewaksbaum ];
```

Comment:
@jbaum98, I'd be happy to add myself as a maintainer here in a followup PR if you'd like a hand.